__tests__/server/middleware/createRequestHtmlFragment.spec.js

```diff
@@ -229,4 +245,86 @@ describe('createRequestHtmlFragment', () => {
     expect(next).toHaveBeenCalled();
     /* eslint-enable no-console */
   });
+
+  it('should open the circuit when event loop lag is > 30ms', async () => {
+    expect.assertions(5);
```

Comment:
is this required with async test ?

Reply:
yes

Reply:
checked locally, `expect.assertions(5);` can be removed.

Reply:
expect.assertions ensures we get passed the promise to all the assertions